test/models/restaurant_test.rb

```diff
@@ -0,0 +1,34 @@
+require 'test_helper'
+
+class RestaurantTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+  test "restaurant attributes should not be empty" do
+    restaurant = Restaurant.new
+    assert restaurant.invalid?
+    assert restaurant.errors[:name].any?
+    assert restaurant.errors[:cuisine].any?
+    assert restaurant.errors[:address].any?
+  end
+
+  test "restaurant rating and max delivery time should be in range" do
+    restaurant = Restaurant.new(name: 'rest 1',
```

Comment:
same note about factories. for example here, you can have the factory build a `Restaurant` instance using default parameters and only override the one you're specifically interested in, e.g. `rating` without having to supply attributes which are of no interest to the test such as `name`